packages/react/src/lib/mergeThemes.ts

```diff
@@ -167,12 +226,22 @@ const mergeThemes = (...themes: ThemeInput[]): ThemePrepared => {
   return themes.reduce<ThemePrepared>(
     (acc: ThemePrepared, next: ThemeInput) => {
       if (!next) return acc
+      const nextDebugId = next['_debugId']
 
-      acc.siteVariables = mergeSiteVariables(acc.siteVariables, next.siteVariables)
+      acc.siteVariables = mergeSiteVariables(
+        acc.siteVariables,
+        withDebugId(next.siteVariables, nextDebugId),
+      )
 
-      acc.componentVariables = mergeThemeVariables(acc.componentVariables, next.componentVariables)
+      acc.componentVariables = mergeThemeVariables(
+        acc.componentVariables,
+        withDebugId(next.componentVariables, nextDebugId),
+      )
 
-      acc.componentStyles = mergeThemeStyles(acc.componentStyles, next.componentStyles)
+      acc.componentStyles = mergeThemeStyles(
+        acc.componentStyles,
+        withDebugId(next.componentStyles, nextDebugId),
```

Comment:
I am not sure that I fully understand why `withDebugId` is required there...

As I understand it will present on each `theme` and then should be propagated to `siteVariables`, `componentVariables`, etc? As I saw, we introduced `createTheme()` function, can we do it there instead?